config/schemas/6/rapidast_schema.json

```diff
@@ -645,6 +645,10 @@
                             "model_type": {
                                 "type": "string"
                             },
+                            "generators": {
+                                "type": "object",
+                                "properties": {}
```

Comment:
this can vary, depending on `model_type`

Reply:
If the `generators` block is empty, this will fail. Do we need to handle this scenario?

Reply:
Do you mean the scenario where the "generators" exist but no properties at all?  If so, I'm not sure if we need to allow that.

Reply:
I agree, I don’t see any value in allowing that. I was wondering, if we allow this scenario in blocks like `general` or `apiScan`, would it make sense here? I don’t fully understand why we allow it in other blocks

Reply:
Oh I see your point. Consistency seems to need to be improved (let's find another chance to do it). just my 2c, for `apiScan` we don't want it to be empty for sure.  For `general`, making it strict would affect the user experience (e.g. a user could configure a 'proxy' and comment the proxy lines out when not used and uncomment, etc).